lib/logcompat.go

```diff
@@ -13,9 +13,3 @@ func logWarn(msg ...interface{}) {
 func logInfo(msg ...interface{}) {
 	log.Println(msg...)
 }
-
-func logDebug(msg ...interface{}) {
```

Comment:
How comes this is being removed?

Reply:
The function isn't used anywhere - and the compiler/linter complains when you have un-used functions. Also we should move to use a proper logging library like zerolog.